ember-cli-build.js

```diff
@@ -36,6 +36,12 @@ module.exports = function(defaults) {
       // runOnPostBuild: (env === 'development') ? 'development-postbuild' : false,
       // shouldActivate: true
     },
+    sassOptions: {
+      includePaths: [
+        'node_modules/bourbon/core',
+        'node_modules/bourbon-neat/app/assets/stylesheets'
```

Comment:
This is how a sass npm package is added to the app. The files become available for `import` same as they are by using the wrapper addon `ember-cli-bourbon`. The addon has an issue which causes the app to fail on build, and this is straightforward enough to warrant removing the addon completely.